frontend/src/components/InlineMessage/InlineMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react'
+import ReactMarkdown from 'react-markdown'
+import { Components } from 'react-markdown/src/ast-to-react'
+import { Box, Flex, Icon, useMultiStyleConfig } from '@chakra-ui/react'
+
+import { BxsErrorCircle, BxsInfoCircle } from '~/assets/icons'
+import { InlineMessageVariant } from '~/theme/components/InlineMessage'
+
+import Link from '../Link'
+
+export type InlineMessageProps = {
+  variant: InlineMessageVariant
+  children: string
+  useMarkdown: boolean
+}
+
+export const InlineMessage = ({
+  variant = 'info',
+  children,
+  useMarkdown = false,
+}: InlineMessageProps): JSX.Element => {
+  const styles = useMultiStyleConfig('InlineMessage', { variant })
+
+  const mdComponents: Components = useMemo(
+    () => ({
+      a: (props) => {
+        return <Link {...props} />
+      },
+    }),
+    [],
+  )
```

Comment:
Seems slightly different from the banner one (banner has external link rendering), should we refactor them into a single hook `hooks/useMdComponents.ts` so the `mdComponents` are consistent between Banner and InlineMessage?

Reply:
added, thanks for going through with me!